### PR TITLE
fix: coordinate example resources

### LINKED ISSUE #484: The example of disallowed/allowed ingress resources in the unique ingress host example has incorrect hostnames
Link: https://open-policy-agent.github.io/gatekeeper-library/website/validation/uniqueingresshost/
The constraint template example uses a simple host_1 == host_2 logic, but the allowed and disallowed examples don't share the same host name -- accordingly the disallowed resource isn't blocked.

example-allowed:
example-allowed-host.example.com
example-allowed-host1.example.com

example-disallowed:
example-host.example.com

example-disallowed2:
example-host2.example.com
example-host3.example.com

### PR DESCRIPTION
**Pull request by JaydipGabani**
**What this PR does / why we need it**: fixes #484 

**Which issue(s) does this PR fix** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by apeabody** [comment]
Hi @JaydipGabani - Can you please take a look at my comment here: https://github.com/open-policy-agent/gatekeeper-library/issues/484#issuecomment-1948976882 Thanks!

File: website/docs/validation/uniqueserviceselector.md [hunk 1]
```diff
@@ -149,11 +149,11 @@ spec:
   ports:
     - port: 443
   selector:
-    key: value
+    key: other-value
 
 ```
 
-Usage
+Usagea
```
Comment (apeabody) [inline website/docs/validation/uniqueserviceselector.md:156]:
```suggestion
Usage
```

**Review by apeabody — CHANGES_REQUESTED** [review]

**Review by maxsmythe — APPROVED** [review]
LGTM

**Review by apeabody — DISMISSED** [review]
Thanks @JaydipGabani! - Per our conversation I would suggest calling this PR to something like "fix: coordinate example resources", or at the very least correct "faulty".

**Comment by JaydipGabani** [comment]
@apeabody this is turning out to be somewhat complex. CI is falling for `ingress-host` because in bats tests `example_allowed` are getting deleted right after the test for them is successful. So by the time when `example_disallowed` are created there is no `example_allowed` resource in the cluster to fail against (this case is being handled with `example_inventory` which we are deleting). If I modify the tests to delete `example_allowed` after the tests are conducted for `example_disallwed`, then other tests fail. Might be worth figuring out how to bring inventory to website directly rather than figuring this out and then eventually bring inventory to website.

wdyt?

**Comment by JaydipGabani** [comment]
Closing this PR in favor of bringing inventory to website.
@maxsmythe @apeabody

**Pull request closed without merging**